turbopack/crates/turbopack/src/lib.rs

```diff
@@ -613,6 +613,10 @@ async fn process_default_internal(
                                 analyze_types,
                                 options,
                             }),
+                            Some(ModuleType::Custom(_)) => {
+                                // TODO
+                                current_module_type
+                            }
```

Comment:
`The module type must be Ecmascript or Typescript to add Ecmascript transforms` because indeed, it is `ModuleType::Custom(RawEcmascriptModuleType)` for the devtools module now